Cargo.toml

```diff
@@ -42,11 +42,11 @@ swc_relay = { version = "0.2.7" }
 testing = { version = "0.33.6" }
 
 # Turbo crates
-turbo-binding = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230504.3" }
+turbo-binding = { package = "turbopack-binding", git = "https://github.com/vercel/turbo.git", branch = "alexkirsz/web-1015-rename-crates" } # last tag: "turbopack-230504.3"
```

Comment:
I thought about doing this, but that's a bigger change that can be made in a different PR.

Also, perhaps we should consider aliasing `turbopack-binding` to just `turbo` to make our code a bit more concise and readable.

Reply:
Indeed. It only works when you `next-link.cjs local`. Otherwise, running the sync script throws with:

```
error: failed to load manifest for workspace member `/Users/alex/dev/nextpack/./next.js/packages/next-swc/crates/core`

Caused by:
  failed to load manifest for dependency `turbopack-binding`

Caused by:
  failed to read `/Users/alex/dev/nextpack/turbo/crates/turbo-binding/Cargo.toml`

Caused by:
  No such file or directory (os error 2)
node:child_process:905
    throw err;
```

Reply:
SGTM!